fr.openfisca.org/fr.openfisca.org.conf

```diff
@@ -41,14 +41,27 @@ server {
         proxy_set_header X-Forwarded-Proto $scheme;
     }
 
+    location /api/latest {
+        proxy_pass http://localhost:9999/;
+        proxy_set_header Host $http_host/api/latest;
+        proxy_http_version 1.1;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+    }
+
     set $served_versions '
     {
       "v21": {
         "href": "https://fr.openfisca.org/api/v21/",
         "shutdownDate": "2019-01-01"
       },
       "v24": {
-        "href": "https://fr.openfisca.org/api/v24/"
+        "href": "https://fr.openfisca.org/api/v24/",
+        "shutdownDate": "2019-07-01"
+      },
+      "lastest": {
+        "href": "https://fr.openfisca.org/api/latest/",
+        "shutdownDate": "tomorrow"
```

Comment:
It's not clear to me what this means in terms of how long I can rely on this API to work.

Reply:
Exactly 24 hours starting from 12AM UTC, or I didn't think searching how to dynamically add a date was worth the effort.

What would you add instead ?

Note that #54 already add to its DOD:

> The doc indicates our public API versioning policy

What we want to communicate here is: if you build your app based on latest today, it'll probably break tomorrow.

Reply:
I might be misunderstanding something, please check my reasoning. At present "latest" is v24, right ? So the shutdownDate of "latest" should be the same as that of v24.

We've also gone from saying that v24 has no planned shutdown, to saying that it will definitely shutdown in a few months' time, without providing a replacement. It seems to me that it would make more sense to wait until v25 is introduced to announce a shutdown date for v24. And I expect that we will then have the same situation as today: no shutdown date announced for v25. So following the line of reasoning above, I would expect to see no shutdown date for "latest" either.

> if you build your app based on latest today, it'll probably break tomorrow.

To me "shutdown" conveys "guaranteed to break", whereas with "probably break" I can take my chances. It's the same deal as capping vs not capping a requirement, if I cap I choose stability, if I go with the latest I choose to deal with the consequences. Bumping a version isn't the same as unpublishing, which is what I take "shutdownDate" to refer to.

Reply:
Yes, `latest` is not `v24`, sorry I didn't comment on that.

Reply:
Wait, so I should've let the shutdown of v24...

Reply:
No strong opinion on that. 
Based on the versioning policy suggested in #54, it is safe to assume that `v24` shutdown date will be `2019-06-01` (or `2020-01-01` if we go for a yearly update). But "waiting until v25 is introduced to announce a shutdown date for v24" would make sense too.

Reply:
I'll let it as such, and open the PR in the doc.

Reply:
Ah, OK. So "latest" means `master`. I guess I could have deduced that from #54 but it's not stated explicitly. Consider this a signal that others might misunderstand too. :)

Reply:
I was also wondering if `/edge` would be more explicit. `/master` could be a candidate too.